src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -413,5 +413,25 @@
     "domain": "www.benefits.va.gov",
     "src": "/pension/current_rates_survivor_pen.asp",
     "dest": "/pension/survivors-pension-rates/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/compensation/current_rates_dic.asp",
+    "dest": "/disability/survivor-dic-rates/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/pension/current_rates_Parents_DIC_pen.asp",
```

Comment:
does it matter if the `"P"` is capitalized here in `Parents`?

Reply:
Good find! It does `redirect.src.toLowerCase()` in the redirect logic so it should be okay 👌